src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -64,6 +65,11 @@ protected function findExtension(string $name)
         $bundles = $this->initializeBundles();
         $minScore = INF;
 
+        $kernel = $this->getApplication()->getKernel();
+        if ($kernel instanceof ExtensionInterface && $kernel instanceof ConfigurationExtensionInterface && $name === $kernel->getAlias()) {
+            return $kernel;
+        }
```

Comment:
A few lines below, we compute a "Did you mean" message. Shouldn't the alias of the kernel be part of it?

Reply:
Sure, done.